contracts/src/v0.8/functions/dev/1_0_0/libraries/FunctionsResponse.sol

```diff
@@ -14,15 +29,15 @@ library FunctionsResponse {
   }
 
   struct Commitment {
-    uint96 adminFee; // -----------┐
+    uint72 adminFee; // -----------┐
     address coordinator; // -------┘
     address client; // ------------┐
     uint64 subscriptionId; //      |
     uint32 callbackGasLimit; // ---┘
-    uint96 estimatedTotalCostJuels; // TODO pack the following
```

Comment:
If I'm not mistaken, if we change the timestamp to uint32 we can reduce this struct by a slot, saving ~25k gas if this is ever stored (not sure if it is since I think we now store hashes? Anyway, free slot if we reorder and reduce timestamp size)

Reply:
```
irb(main):001:0> Time.at(2**32)
=> 2106-02-07 01:28:16 -0500
```

Reply:
Looks like this saved `1978` gas :+1: